statsmodels/base/model.py

```diff
@@ -745,7 +746,7 @@ def fit(self, start_params=None, method='nm', maxiter=500, full_output=1,
     #fit.__doc__ += LikelihoodModel.fit.__doc__
 
 
-class Results(object):
+class Results(dimensions.NEQsMixin, dimensions.NobsMixin):
```

Comment:
the same mixin will not work for Model and Results
a large number of test failures, break early because results instances don't have endog
`AttributeError: 'ARResults' object has no attribute 'endog'`